script/download_rust_deps.py

```diff
@@ -34,7 +35,7 @@ def GetUrl(platform):
 
 
 def AlreadyUpToDate():
-    if not os.path.exists(RUSTUP_DIR):
+    if not os.path.exists(RUST_DEPS_DIR):
```

Comment:
Have you already updated the files on S3?

Reply:
👍